certificates/Milliskilriki.cer

```diff
@@ -1,32 +1,30 @@
-c3JvdC5pcy9pc2xhbmRzcm90L2xhdGVzdC5jcmwwHQYDVR0OBBYEFMIpPob/hsTa
-NR9ppqT/AYM8SjOpMA0GCSqGSIb3DQEBCwUAA4ICAQBOBaAGxLPntnC3fVeHeQsD
-13AT2a3+Ry9R3AFP063Cwnc4+SYlCt+LSPa2wTCcDXAjvPgaxjRrTuTF1sCfj1uu
-8GVMeo7e6suYggf2PLxsKWjaGOJhUtZnNC1rh2mt4TVYTR2D0AHYju0nNjzZlXU3
-1Ea//HDCQkV9+sSINTTrFL0Y5kB05WyVBXLHSTl/bKY8ULik2JImofrF+nI7GifH
-CMLFkCOkAUDsI3Fd0Fh7v3NguxpOM4sov2jowMZzxqkS9B8B0qRO41h6spuLvsNS
-tnFZHIbTrMaINUm9X6C49lr0fpRri4UNQa2prMgNsK9dwsYurlMga1WpO6fwkzU3
-mLYjitUxV9iYJ1VWj2jhJt0ofDsB4xLCVu8n0gekde09P5EdWzLvXD03PLtkEiGt
-HElpluMFYaFjHhofYhai3u5eLVFTcNkEcyZO470EZTZ123dP3JQpSBKFjH7Z5CSS
-wgvFB/zOOdnEDS49Iidetk5y2D8Pg2NB4qGJcmkit9Zjv2cOj9b5gKu7XMzkja9v
-MahgfOrAKZJS7nrxr6NNR4rTMw8Rb9nvkS2sGk9ZGnbMLM+j84LDpT0rFMLNkegD
-FVob63pewil6mH43mVOLq6tyYMiU7mgzYcui1rGnhtLiSqpI6L1UaQ2IqT4PJGdX
-PIeO6oWVJpZaF7hWABD7sw==
------END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIF4zCCBMugAwIBAgIDJRaUMA0GCSqGSIb3DQEBCwUAMH4xCzAJBgNVBAYTAklTMRMwEQYDVQQF
+Ewo1MjEwMDAyNzkwMRUwEwYDVQQKEwxBdWRrZW5uaSBoZi4xJzAlBgNVBAsTHlV0Z2VmYW5kaSBm
+dWxsZ2lsZHJhIHNraWxyaWtqYTEaMBgGA1UEAxMRRnVsbGdpbHQgYXVka2VubmkwHhcNMjIwNTI0
+MTE1NzEyWhcNMjYwNTI0MTE1NzEyWjCBuTEXMBUGA1UECxMOMjAyMjA1MTYxMjMzMjAxHjAcBgNV
+BAoMFcOeasOzw7Bza3LDoSDDjXNsYW5kczELMAkGA1UEBhMCSVMxGDAWBgNVBAsTD0J1bmFkYXJz
+a2lscmlraTEiMCAGA1UECwwZQXXDsGtlbm5pbmcgb2cgdW5kaXJyaXR1bjETMBEGA1UEBRMKNjUw
+Mzc2MDY0OTEeMBwGA1UEAxMVSW5uc2tyYW5pbmcgSXNsYW5kLmlzMIIBIjANBgkqhkiG9w0BAQEF
+AAOCAQ8AMIIBCgKCAQEAj4iGMAvUYla7kJQ4l3xm/sday7+jQeTjjvRha0P5hrzooHEfiV2CDKlp
+cRWSdPUF7aM/kEnoSUuh23EuHJ3u1/pzgBoR+f3eaw0aOmJWP+5BtEqJ7ianQL9115XVgs8la0sr
+/RWj38q2Ft+3EKWjLioHyRa4OhIdCCk5xL6rZcg1MoOvVVPJdcX8MMpr6UNHaKQcBEriYNnJP4os
+yUaVT5SWaEVaewzft1jFuugsvJFQsPnLoVq8KLnF4R8iPGUF2nIlRShspO1exX7NuiIBvz210jf3
+PyfpoqFZ8Bjt7YTXqXZ0xdSSWmLm3aCNZ0kCabg5F1FYYY0OJ9Z8m33orQIDAQABo4ICLDCCAigw
+DAYDVR0TAQH/BAIwADCCAQsGA1UdIASCAQIwgf8wgfwGCWCCYAECAQEBAjCB7jCBxAYIKwYBBQUH
+AgIwgbcagbRUaGlzIGNlcnRpZmljYXRlIGlzIGludGVuZGVkIGZvciBkaWdpdGFsIHNpZ25hdHVy
+ZXMgYW5kIGF1dGhlbnRpY2F0aW9uLiBUaGlzIGNlcnRpZmljYXRlIGZ1bGZpbHMgdGhlIHJlcXVp
+cmVtZW50cyBvZiBub3JtYWxpemVkIGNlcnRpZmljYXRlIHBvbGljeSAoTkNQKSBkZWZpbmVkIGlu
+IEVUU0kgVFMgMTAyIDA0Mi4wJQYIKwYBBQUHAgEWGWh0dHBzOi8vcmVwby5hdWRrZW5uaS5pcy8w
+dwYIKwYBBQUHAQEEazBpMCMGCCsGAQUFBzABhhdodHRwOi8vb2NzcC5hdWRrZW5uaS5pczBCBggr
+BgEFBQcwAoY2aHR0cDovL2NkcC5pc2xhbmRzcm90LmlzL3NraWxyaWtpL2Z1bGxnaWx0YXVka2Vu
+bmkucDdiMAsGA1UdDwQEAwIF4DAfBgNVHSMEGDAWgBTCKT6G/4bE2jUfaaak/wGDPEozqTBDBgNV
+HR8EPDA6MDigNqA0hjJodHRwOi8vY3JsLmF1ZGtlbm5pLmlzL2Z1bGxnaWx0YXVka2VubmkvbGF0
+ZXN0LmNybDAdBgNVHQ4EFgQU0SOqN/DGELKdA++dg68quf14bNgwDQYJKoZIhvcNAQELBQADggEB
+AMiangTaGuXvtF/S5RYjEBLIqdknzRjwDA485w4lSisjKAGBN/6omUyj5h0Y6fCSDXyC0EeBdzVt
+CmdZl0DeKMpBiEP0SoL2t7xclrNZ+KBTqSZiD0Y7xBLVr0haKFi+ruJqRCHk8BNpV59M1XD121Fv
+KedLDLDu/Q4yq1GWo3ubEX2ezOoAgGYAfaarSYaXp4W/d2jJ3bMVsPCyYkj9JGZSNzHg68WhL7dU
+1RbQKSotibHUGVWNcwDnvcOovWAe4k5lWm/QtX+TwwWOtFaKmk7pok5jeIiiWIU+YSp8skkUCLwD
+bfDWYPFtBWkN4hQYQkg52Jyu6g90DAn19zgl/tM=
+

```

Comment:
Can you remove the empty line since openssl can't parse it when it's there

Reply:
Sure thing, thank you @sunkan